preupg/xmlgen/oscap_group_xml.py

```diff
@@ -55,33 +55,28 @@ def find_all_ini(self):
         for file_name in self.lists:
             if FileHelper.check_file(file_name, "r") is False:
                 continue
-            try:
-                config = configparser.ConfigParser()
-                filehander = codecs.open(file_name, 'r', encoding=settings.defenc)
-                config.readfp(filehander)
-                fields = {}
-                if config.has_section('premigrate'):
-                    section = 'premigrate'
-                else:
-                    section = 'preupgrade'
-                for option in config.options(section):
-                    fields[option] = config.get(section, option)
-                self.loaded[file_name] = [fields]
-            except configparser.MissingSectionHeaderError:
-                MessageHelper.print_error_msg(title="Missing section header")
-            except configparser.NoSectionError:
-                MessageHelper.print_error_msg(title="Missing section header")
-            except configparser.ParsingError:
-                MessageHelper.print_error_msg(title="Incorrect INI file\n", msg=file_name)
-                os.sys.exit(1)
+            config = configparser.ConfigParser()
+            filehander = codecs.open(file_name, 'r', encoding=settings.defenc)
+            config.readfp(filehander)
+            fields = {}
+            if config.has_section('premigrate'):
+                section = 'premigrate'
+            else:
+                section = 'preupgrade'
+            for option in config.options(section):
+                fields[option] = config.get(section, option)
+            self.loaded[file_name] = [fields]
```

Comment:
There is a code in _check_recommended_fields_ that makes no sense, that causes this issue. It does not raise the correct exception when some of the required fields ['content_title', 'check_script', 'solution'] is missing AND recommended field 'applies_to' is missing in INI.
I've fixed it in https://github.com/upgrades-migrations/preupgrade-assistant/pull/230/files#diff-9eae4399e4b933991c4b81856022426dL169